test/functionalTest/cases/3162/registration_without_provider.test

```diff
@@ -0,0 +1,82 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
PR shouldn't contain additional .test, isn't it? Maybe you git-added too much before commit? ;)

Reply:
Yeah, that's a mistake. Not supposed to add any test. Removing it

Reply:
Fixed in a703b34